bears/natural_language/AlexBear.py

```diff
@@ -18,6 +21,30 @@ class AlexBear:
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
 
+    @classmethod
+    def check_prerequisites(cls):  # pragma: no cover
+        parent_prereqs = super().check_prerequisites()
+        if parent_prereqs is not True:
+            return parent_prereqs
+
+        incorrect_pkg_msg = (
+            "Please ensure that the package that has been installed is the "
+            "one to 'Catch insensitive, inconsiderate writing'. This can be "
+            "verified by running `alex --help` and seeing what it does.")
+        try:
+            output = subprocess.check_output(("alex", "--help"),
+                                             stderr=subprocess.STDOUT)
+        except (OSError, subprocess.CalledProcessError):
+            return ("The `alex` package could not be verified. " +
+                    incorrect_pkg_msg)
+        else:
+            output = output.decode(sys.getfilesystemencoding())
+            if "Catch insensitive, inconsiderate writing" in output:
```

Comment:
I find the current method to be cleaner as compared to following symlinks.
In a distributed environment like NFS/HDFS symlinks aren't well understood and errors out. Just yesterday I was writing a Python+Spark script which couldn't understand symlinks due to various FileSystem/configuration issues. But I guess npm itself would install things incorrectly in such cases ^_^ and AlexBear wouldn't work.
Secondly, if you're using a non-typical npm installation, you have these wrapper scripts that call other scripts (without symlinks). Something like how pyenv and setuptools does. Or if you're using a bash alias, that method will always fail.

Is there any downside of the current way ? Is there a plan to change the slogan in the help ?